pool/app/message_template/default/locales/default_en.ml

```diff
@@ -312,7 +317,10 @@ let session_cancellation =
   let sms_text =
     {|The following session you have registered to was canceled:
 
-{sessionOverview}|}
+{sessionOverview}
+
+Reason: {reason}|}
```

Comment:
Does a reason has to be added to the cancellation?
Otherwise, "Reason:" should only be added, if there is one.

Reply:
Yes, a reason has to be provided.

Reply:
@mabiede ping